scripts/core/kernel.yml

```diff
@@ -534,3 +534,45 @@ params:
 returns:
     - $X_RESULT_ERROR_UNSUPPORTED_FEATURE:
         - "If the adapter has no underlying equivalent handle."
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get the suggested local work size for a kernel from runtime implementation."
```

Comment:
Do we need `from runtime implementation`? Most of other ur functions will also be lowered to runtime implementation, right?

Reply:
Yeah, that makes sense. I will remove that.